addons/source-python/packages/source-python/players/engines/orangebox/tf.py

```diff
@@ -0,0 +1,49 @@
+# ../players/engines/orangebox/cstrike.py
+
+"""Provides Team Fortress 2 specific Player based functionality."""
+
+# =============================================================================
+# >> IMPORTS
+# =============================================================================
+# Source.Python Imports
+from . import Player as _Player
+from configobj import ConfigObj
+from core import GAME_NAME
+from paths import SP_DATA_PATH
+
+# =============================================================================
+# >> CLASSES
+# =============================================================================
+class Player(_Player):
+    caching = True
+    
+    # Team
+    _team_data = ConfigObj(SP_DATA_PATH / 'teams' / GAME_NAME + '.ini', unrepr=True)
+
+    _teams_names = _team_data.get('names')
```

Comment:
This mapping is already available as [players.teams.teams_by_name](https://github.com/Source-Python-Dev-Team/Source.Python/blob/master/addons/source-python/packages/source-python/players/teams.py#L35-L36) and should really just be imported from there rather than queried again from the INI file.